### PR TITLE
refactor(es/minifier): Remove `mangle.safari10`

### PR DESCRIPTION
**Pull request by Auspicus**
When the `mangle.safari10` option is set to true identifiers are re-used in the `try/catch` block which results in a parse error.

**Description:**

Using the `mangle.safari10` option causes identifiers to be re-used in a `try/catch` block.

**BREAKING CHANGE:**

**Related issue (if exists):**
https://github.com/swc-project/swc/issues/8837

### CODE REVIEW

File: crates/swc_ecma_transforms_base/src/rename/mod.rs [hunk 1]
```diff
@@ -130,7 +130,7 @@ where
     {
         let mut scope = {
             let mut v = Analyzer {
-                safari_10: self.config.safari_10,
+                safari_10: false,
```
Comment (Auspicus) [inline crates/swc_ecma_transforms_base/src/rename/mod.rs:133]: Can use `Default::default()` here?

**Comment by kdy1** [comment]
You can do `UPDATE=1 cargo test` from `./crates/swc_ecma_minifier` to update test snapshot files

**Comment by Auspicus** [comment]
@kdy1 I think we'll have divergence with the terser output for the `broken_safari_catch_scope` test if we remove this option. Is that OK?

**Comment by kdy1** [comment]
Yeap, it's fine

**Review by kdy1 — APPROVED** [review]
swc-bump:
 - swc_ecma_transforms_base
 - swc_ecma_minifier
 - swc_core

**Comment by kdy1** [comment]
Can you run `UPDATE=1 cargo test --test projects --test tsc` from `./crates/swc`?

**Pull request merged**